net/shadowsocks/src/opnsense/mvc/app/models/OPNsense/Shadowsocks/General.xml

```diff
@@ -31,5 +30,21 @@
             <default>password</default>
             <Required>N</Required>
         </password>
+        <cipher type="OptionField">
+            <default>aes-256-cfb</default>
```

Comment:
Any reason for the change of the default? Just curious, looks sane enough.

Reply:
Read that chacha ist most secure, man page says default is RC4 oO 
Since Laven is using aes-256-cfb and this seems to be a most compatible I chosed this one.

Reply:
👍